### PR TITLE
Correção das formulas de algumas magias.

### PR DESCRIPTION
**Pull request by morenao97**
Corrigi as formulas de dano maximo e minimo de algumas magias, usando como base o Tibia Wikia. (https://tibia.fandom.com/wiki/Formulae)
```
Magias Corrigidas:
    Healing:
        Light Healing
        Intense Healing
        Wound Cleasing
        Mass Healing
        Ultimate Healing
        Divine Healing
        Heal Friend

    Instant Attack
        Energy Beam
        Great Energy Beam
        Divine Caldera
        Terra Wave
        Energy Wave
        Rage of Skies
        Hell's Core
        Wrath of Nature
        Eternal Winter
        Energy Strike
        Terra Strike
        Ice Strike
        Physical Strike
        Flame Strike
        Death Strike
        Divine Missile
        Ice Wave
        Fire Wave
        Whirlwind throw
        Groundshaker
        Berserk
        Fierce Berserk
        Etheral Spear

    Runes:
        Explosion
        LMM
        HMM
        Stalagmite
        Fireball
        Icicle
        Holy Missile
        Sudden Death
        Thunderstorm
        Stone Shower
        Avalanche
        Great Fireball
```

### CODE REVIEW

File: data/spells/scripts/attack/avalanche.lua [hunk 1]
```diff
@@ -5,8 +5,8 @@ combat:setParameter(COMBAT_PARAM_DISTANCEEFFECT, CONST_ANI_ICE)
 combat:setArea(createCombatArea(AREA_CIRCLE3X3))
 
 function onGetFormulaValues(player, level, magicLevel)
```
Comment (majestyotbr) [inline data/spells/scripts/attack/avalanche.lua]: maglevel

File: data/spells/scripts/attack/energy wave.lua [hunk 2]
```diff
@@ -5,8 +5,8 @@ combat:setParameter(COMBAT_PARAM_DISTANCEEFFECT, CONST_ANI_ENERGY)
 combat:setArea(createCombatArea(AREA_SQUAREWAVE5, AREADIAGONAL_SQUAREWAVE5))
 
 function onGetFormulaValues(player, level, magicLevel)
```
Comment (majestyotbr) [inline data/spells/scripts/attack/energy wave.lua]: maglevel

File: data/spells/scripts/attack/eternal winter.lua [hunk 3]
```diff
@@ -4,8 +4,8 @@ combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_ICETORNADO)
 combat:setArea(createCombatArea(AREA_CIRCLE5X5))
 
 function onGetFormulaValues(player, level, magicLevel)
```
Comment (majestyotbr) [inline data/spells/scripts/attack/eternal winter.lua]: maglevel

File: data/spells/scripts/attack/hells core.lua [hunk 4]
```diff
@@ -4,8 +4,8 @@ combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_FIREAREA)
 combat:setArea(createCombatArea(AREA_CIRCLE5X5))
 
 function onGetFormulaValues(player, level, magicLevel)
```
Comment (majestyotbr) [inline data/spells/scripts/attack/hells core.lua]: maglevel

File: data/spells/scripts/attack/rage of the skies.lua [hunk 5]
```diff
@@ -4,8 +4,8 @@ combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_BIGCLOUDS)
 combat:setArea(createCombatArea(AREA_CIRCLE6X6))
 
 function onGetFormulaValues(player, level, magicLevel)
```
Comment (majestyotbr) [inline data/spells/scripts/attack/rage of the skies.lua]: maglevel

File: data/spells/scripts/attack/wrath of nature.lua [hunk 6]
```diff
@@ -4,8 +4,8 @@ combat:setParameter(COMBAT_PARAM_EFFECT, CONST_ME_SMALLPLANTS)
 combat:setArea(createCombatArea(AREA_CIRCLE6X6))
 
 function onGetFormulaValues(player, level, magicLevel)
```
Comment (majestyotbr) [inline data/spells/scripts/attack/wrath of nature.lua]: maglevel

**Comment by morenao97** [comment]
Corrigido.

**Review by majestyotbr — APPROVED** [review]

File: data/spells/scripts/attack/berserk.lua [hunk 7]
```diff
@@ -6,9 +6,12 @@ combat:setParameter(COMBAT_PARAM_USECHARGES, 1)
 combat:setArea(createCombatArea(AREA_SQUARE1X1))
 
 function onGetFormulaValues(player, skill, attack, factor)
-	local skillTotal = skill * attack
-	local levelTotal = player:getLevel() / 5
-	return -(((skillTotal * 0.07) + 7) + (levelTotal)), -(((skillTotal * 0.09) + 11) + (levelTotal))
+	local level = player:getLevel()
+	
+	local min = (level / 5) + ( skill + attack )*0.5
```
Comment (majestyotbr) [inline data/spells/scripts/attack/berserk.lua]: local min = (level / 5) + (skill + attack) * 0.5

File: data/spells/scripts/attack/berserk.lua [hunk 8]
```diff
@@ -6,9 +6,12 @@ combat:setParameter(COMBAT_PARAM_USECHARGES, 1)
...
+	local max = (level / 5) + (skill + attack)*1.5
```
Comment (majestyotbr) [inline data/spells/scripts/attack/berserk.lua]: local max = (level / 5) + (skill + attack) * 1.5

File: data/spells/scripts/attack/ethereal spear.lua [hunk 9]
```diff
@@ -5,8 +5,12 @@ combat:setParameter(COMBAT_PARAM_DISTANCEEFFECT, CONST_ANI_ETHEREALSPEAR)
 combat:setParameter(COMBAT_PARAM_BLOCKARMOR, 1)
 
 function onGetFormulaValues(player, skill, attack, factor)
-	local levelTotal = player:getLevel() / 5
-	return -(((2 * skill + attack / 3500) * 0.35) + (levelTotal) + 0), -(((2 * skill + attack / 3125) * 0.5) + (levelTotal) + 5)
+	local level = player:getLevel()
+	
+	local min = (level / 5) + ( skill + 25 )/3
```
Comment (majestyotbr) [inline data/spells/scripts/attack/ethereal spear.lua]: local min = (level / 5) + (skill + 25)  / 3

File: data/spells/scripts/attack/fierce berserk.lua [hunk 10]
```diff
@@ -6,11 +6,15 @@ combat:setParameter(COMBAT_PARAM_USECHARGES, 1)
 combat:setArea(createCombatArea(AREA_SQUARE1X1))
 
 function onGetFormulaValues(player, skill, attack, factor)
-	local skillTotal = skill * attack
-	local levelTotal = player:getLevel() / 5
-	return -(((skillTotal * 0.12) + 12) + (levelTotal)), -(((skillTotal * 0.15) + 27) + (levelTotal))
+	local level = player:getLevel()
+	
+	local min = (level / 5) + ( skill + 2*attack )*1.1
```
Comment (majestyotbr) [inline data/spells/scripts/attack/fierce berserk.lua]: local min = (level / 5) + (skill + 2 * attack ) * 1.1

File: data/spells/scripts/attack/fierce berserk.lua [hunk 11]
```diff
@@ -6,11 +6,15 @@ combat:setParameter(COMBAT_PARAM_USECHARGES, 1)
...
+	local max = (level / 5) + (skill + 2*attack)*3
```
Comment (majestyotbr) [inline data/spells/scripts/attack/fierce berserk.lua]: local max = (level / 5) + (skill + 2 * attack) * 3

File: data/spells/scripts/attack/groundshaker.lua [hunk 12]
```diff
@@ -6,9 +6,12 @@ combat:setParameter(COMBAT_PARAM_USECHARGES, 1)
 combat:setArea(createCombatArea(AREA_CIRCLE3X3))
 
 function onGetFormulaValues(player, skill, attack, factor)
-	local skillTotal = skill * attack
-	local levelTotal = player:getLevel() / 5
-	return -(((skillTotal * 0.02) + 4) + (levelTotal)), -(((skillTotal * 0.03) + 6) + (levelTotal))
+	local level = player:getLevel()
+	
+	local min = (level / 5) + ( skill + attack )*0.5
```
Comment (majestyotbr) [inline data/spells/scripts/attack/groundshaker.lua]: local min = (level / 5) + (skill + attack) * 0.5

File: data/spells/scripts/attack/groundshaker.lua [hunk 13]
```diff
@@ -6,9 +6,12 @@ combat:setParameter(COMBAT_PARAM_USECHARGES, 1)
...
+	local max = (level / 5) + (skill + attack)*1.1
```
Comment (majestyotbr) [inline data/spells/scripts/attack/groundshaker.lua]: local max = (level / 5) + (skill + attack) * 1.1

File: data/spells/scripts/attack/whirlwind throw.lua [hunk 14]
```diff
@@ -6,9 +6,12 @@ combat:setParameter(COMBAT_PARAM_BLOCKARMOR, 1)
 combat:setParameter(COMBAT_PARAM_USECHARGES, 1)
 
 function onGetFormulaValues(player, skill, attack, factor)
-	local skillTotal = skill * attack
-	local levelTotal = player:getLevel() / 5
-	return -(((skillTotal * 0.01) + 1) + (levelTotal)), -(((skillTotal * 0.03) + 6) + (levelTotal))
+	local level = player:getLevel()
+	
+	local min = (level / 5) + ( skill + attack )/3
```
Comment (majestyotbr) [inline data/spells/scripts/attack/whirlwind throw.lua]: local min = (level / 5) + (skill + attack) / 3

**Pull request merged**